.gitignore

```diff
@@ -25,4 +25,4 @@ compile_commands.json
 .vagrant
 
 # binaries and build files
-dist
\ No newline at end of file
+dist
```

Comment:
Touched this file when created the syscaller tool in the implementation first stage. So, just a not related formatting fix here.